packages/bpk-component-datatable/src/utils.js

```diff
@@ -21,3 +21,13 @@ export const hasClassName = (node, className) => {
   const nodeClassName = node.getAttribute('class');
   return nodeClassName && nodeClassName.split(' ').indexOf(className) !== -1;
 };
+
+export const remToPx = value => {
+  let parsed = null;
+
+  if (/rem$/.test(value)) {
+    parsed = parseFloat(value.replace(/rem/, '')) * 16;
```

Comment:
This does not actually work because there is no guarantee that the root font size is 16. This will break when zooming for example. [This PR](https://github.com/Skyscanner/backpack/pull/877/files) is an example where we fixed an issue based on an assumption like this

Reply:
Given this function was used only in the test, I think it makes more sense to just use a number instead of the `spacingSm` token, so I've removed this function again and replaced the token in the test.

What do you think @k0nserv ?